From 9b75010faec8403f095f175ef7977ad6a2d960cf Mon Sep 17 00:00:00 2001
Message-Id: <9b75010faec8403f095f175ef7977ad6a2d960cf.1413837857.git.chang-joon.lee@intel.com>
In-Reply-To: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
References: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
From: Rafael Barbalho <rafael.barbalho@intel.com>
Date: Thu, 25 Sep 2014 11:47:04 +0100
Subject: [PATCH 295/312] FOR_UPSTREAM [VPG]: drm/i915: Remove unused variable
 in rps code

The following commit left an unused variable in cherrview_disable_rps:
Author: Deepak S <deepak.s@intel.com>
Date:   Thu Jul 24 15:17:05 2014 +0530

    FOR_UPSTREAM [VPG]: drm/i915: Add rc6 Debugfs support to CHV

For: GMINL-1778
Change-Id: Ieedd0de6aaccda37d9b69d719ff8d64813bc4c53
Signed-off-by: Rafael Barbalho <rafael.barbalho@intel.com>
---
 drivers/gpu/drm/i915/intel_pm.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_pm.c b/drivers/gpu/drm/i915/intel_pm.c
index 10d87c5..6fdc722 100644
--- a/drivers/gpu/drm/i915/intel_pm.c
+++ b/drivers/gpu/drm/i915/intel_pm.c
@@ -3939,8 +3939,6 @@ static void gen6_disable_rps(struct drm_device *dev)
 
 static void cherryview_disable_rps(struct drm_device *dev)
 {
-	struct drm_i915_private *dev_priv = dev->dev_private;
-
 	/* Disable rc6 */
 	vlv_set_rc6_mode(dev, true);
 
-- 
1.7.9.5

